### PR TITLE
March build docs

### PR DESCRIPTION
**Pull request by mmroden**
Built out the next version of the docs, including testing docs and assembly drawings.

### CODE REVIEW

File: content/en/build-guide/_index.adoc [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
 linkTitle: "PAPRa Build Guide"
 weight: 1
 description: >
-  How to build the January 31 2021 version of the Tetra PAPR.
+  How to build the March 2021 version of the Tetra PAPR.
 ---
 
 == Quick Synopsis
 
 This guide will walk you through how to build a Personal Air Purifying Respirator, also known as a PAPR.  We assume that you have some knowledge of 3D-printing, and cutting foam with a knife or using a stencil or a cricut, and perhaps some soldering.  In the end, you will have four things: a box with a filter and a fan in it, a second box with batteries and a fan controller, a mask, and straps and hoses to hold it all together.
```
Comment (capsulecorplab) [inline content/en/build-guide/_index.adoc]: _Personal_ should say _Powered_

File: content/en/build-guide/_index.adoc [hunk 2]
```diff
@@ -1,31 +1,33 @@
...
 
-Please note that this system will not protect you from anything landing in your eyes.  For that protection, either the mask will need to be modified to be a helmet, or you will need a face shield.
+Please note that this system will not protect you from anything landing in your eyes.  For that protection, either the mask will need to be modified to be a helmet or you will need a face shield.
+
+An important point: we provide no warranty, implicit or explicit, that this device will work up to an N95 specification, when built outside of a facility rated to do so.  The only way to truly test one of these devices using something like a https://tsi.com/products/respirator-fit-testers/portacount-respirator-fit-tester-8038/[Portacount] under controlled conditions.
```
Comment (capsulecorplab) [inline content/en/build-guide/_index.adoc]: Personally, I would convert this text block to an [admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/), replacing _An important point_ with **CAUTION**, but it's purely an aesthetic choice.

File: content/en/testing-guide/_index.adoc [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+---
+
+== Quick Synopsis
+
+This page will walk you through how we established that the PAPRa can offer N95 levels of protection on a mask when worn by someone with a beard, and N99 levels of protection by the fan box itself.  These tests are by no means exhaustive; far more testing is needed to ensure that these results can be achieved consistently, and the testing should be done in a more controlled environment than what we have done.
+
+Furthermore, we offer no warranty, explicit or implied, that any device that you create will achieve similar standards of filtration.  We are currently working to determine the best path forward for individuals who make their PAPRs to test them.
+
+== Parts Needed
+
```
Comment (capsulecorplab) [inline content/en/testing-guide/_index.adoc:22]: Should there be a tabulated bill of materials (for testing) here?

File: content/en/testing-guide/_index.adoc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+In addition to the PAPR to be tested, we tested using a https://tsi.com/products/respirator-fit-testers/portacount-respirator-fit-tester-8038/[Portacount] with a hose assembly as depicted below:
+
+[link=https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-N5dwqjg/0/a18cf5f4/5K/_MMR0431-5K.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/PAPRa-Build-13-March-2021/i-N5dwqjg/0/a18cf5f4/L/_MMR0431-L.jpg[] 
+
+It is important to note that the Portacount is effective at showing the isolation of airborne particles, but not at identifying the sizes of particles measured, and did not have a setting for an N100 level of protection.  When testing for N95 levels of protection, the Portacount has a range from 0 to 200+, with 100 passing, while N99 testing has a range from 0 to 2000+, with 500 passing.  All tests depicted in this document were done using online testing, waiting several seconds to ensure that the readings were stable before recording them.
```
Comment (capsulecorplab) [inline content/en/testing-guide/_index.adoc]: I'd also convert this text block to an [admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/), prepended with the **NOTE** keyword

File: content/en/build-guide/fan-box.adoc [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+weight: 1
+description: >
+  How-To guide for building the Tetra PAPRa fan box (March 2021 version).
+---
+
+== Quick Synopsis ==
+
+The Tetra Powered Air Purified Respirator requires that the air be purified; this purification is accomplished by powering a fan to pull air through a filter and then pushing that purified air through a hose to a mask worn on the face.  This section contains the instructions for building the box that holds the fan and filter.
+
+Please note that this version of the fanbox has a known problem-- the power connector specified on the PCB is proprietary, and does not properly link into the provided DC cables.  The next version of the fan box will fix this problem.
```
Comment (capsulecorplab) [inline content/en/build-guide/fan-box.adoc]: I'd also convert this text block to an [admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/), prepended with the **NOTE** keyword

**Review by capsulecorplab — APPROVED** [review]
Some review notes for styling-related improvements. Otherwise, LGTM

File: content/en/testing-guide/_index.adoc (hunk 3 above)
Comment (mmroden) [reply content/en/testing-guide/_index.adoc:22]: Yes, but I don't have one

Comment (capsulecorplab) [reply content/en/testing-guide/_index.adoc:22]: opened an [issue](https://github.com/tetrabiodistributed/papra/issues/11)

**Pull request merged**